charts/flink-job/templates/rocksdb-pvc.yaml

```diff
@@ -0,0 +1,15 @@
+{{- if .Values.pvcRocksdb.enabled -}}
```

Comment:
So - in this case, I don't think this is right. You're not creating a single PVC, but you actually want to create a PVC for each and every Pod that runs in the StatefulSet that the Flink Operator is managing. Your going to want something like https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/#components - see the `volumeClaimTemplate` setting. There is a discussion about this exact scenario at https://github.com/GoogleCloudPlatform/flink-on-k8s-operator/issues/353.

@sabw8217 When I read https://github.com/shashken/flink-on-k8s-operator/blob/master/docs/crd_v1alpha1.md, it is not immediately clear to me that they support setting up the `volumeClaimTemplate` setting. Am I missing something obvious?

Reply:
That's not the CRD we are using.

https://github.com/GoogleCloudPlatform/flink-on-k8s-operator/blob/master/docs/crd.md

Reply:
but @diranged is correct, you do not need this. The PVC is generated for you from the template in the StatefulSet, you do not need to create it separately.

Reply:
🤦  thanks aaron.. oops..that CRD does have the volumeClaimTemplates section..